### DISCUSSION TITLE
Does anyone know how to fix the icon position?

### DISCUSSION THREAD

**Discussion by Screamliner** [eduardofilo/RG350_adam_image#349] [Q&A]
![IMG_3774.jpg](https://github.com/eduardofilo/RG350_adam_image/assets/124108128/1cea68b0-4fb8-4af8-a308-bff24a289527)

adam image 2.1 rg300x

**Comment by a-dekker** [comment]
https://github.com/OpenDingux/gmenu2x/issues/28

**Reply by Screamliner** [reply]
Thanks for the help.  But where to change y-7 to y-22?

**Reply by a-dekker** [reply]
That is mentioned in the issue I created.
But unfortunately it is not a matter of changing some config file yourself, this is a change in the source code, so it needs to be compiled and shipped with the OpenDingux installation. For that reason I created the issue over there.

**Reply by Screamliner** [reply]
As I thought, it's not something I can change.  Anyway thanks for your help.

**Reply by a-dekker** [reply]
Some settings are configurable, like the height of of the lower bar. That can be configured in the skin.conf from your used skin. But that does not fix this issue.
I changed the menu program 'gmenu2x' itself. But that program is put into a readonly squashfs image. You can update that image in theory, but it is a lot of work and will be overwritten with the next update. so it should be handled upstream.

This is how the change worked out for me btw:
before:
![screenshot003](https://github.com/eduardofilo/RG350_adam_image/assets/10128456/8fed0e3a-78dc-4c01-be49-8fbf791d3f3d)

after:
![screenshot004](https://github.com/eduardofilo/RG350_adam_image/assets/10128456/c859613c-3556-4e14-9fd2-6289a1b84f61)

Let's hope the OpenDingux developers pick up the issue!

**Reply by Screamliner** [reply]
It's great that the icons are in the right place.  It's an outdated device, and most people like simplemenu, so I don't hold out much hope that it will be fixed.

**Reply by a-dekker** [reply]
True, the original gmenu2x repository is archived and OpenDingux is mainly focusing on keeping the kernel up-to-date. And indeed the Adam image is focused around the use of SimpleMenu.

**Reply by Screamliner** [reply]
By the way , a rg280v user told me 280v doesnt have this problem.

**Reply by a-dekker** [reply]
Ah, then I think it is related to the screen resolution, as your RG300 has the same 640x480 screen size as my RG350M.
